fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/api/LoanChargesApiResource.java

```diff
@@ -224,13 +220,16 @@ public String executeLoanCharge(@PathParam("loanId") @Parameter(description = "l
             @Parameter(hidden = true) final String apiRequestBodyAsJson) {
 
         final CommandWrapperBuilder builder = new CommandWrapperBuilder().withJson(apiRequestBodyAsJson);
-        CommandProcessingResult result = null;
-        if (is(commandParam, "waive")) {
+        CommandProcessingResult result;
+        if (CommandParameterUtil.is(commandParam, "waive")) {
             final CommandWrapper commandRequest = builder.waiveLoanCharge(loanId, loanChargeId).build();
             result = this.commandsSourceWritePlatformService.logCommandSource(commandRequest);
-        } else if (is(commandParam, "pay")) {
+        } else if (CommandParameterUtil.is(commandParam, "pay")) {
             final CommandWrapper commandRequest = builder.payLoanCharge(loanId, loanChargeId).build();
             result = this.commandsSourceWritePlatformService.logCommandSource(commandRequest);
+        } else if (CommandParameterUtil.is(commandParam, "adjustment")) {
```

Comment:
Slight cleanup but can you pls extract these magic strings into constants in this class? Thanks.

Reply:
sure